src/libmongoc/CMakeLists.txt

```diff
@@ -856,35 +856,67 @@ if (ENABLE_SHM_COUNTERS STREQUAL "ON")
    )
 endif ()
 
+cmake_dependent_option (
+   MONGOC_TEST_USE_CSFLE "If TRUE, download and use a csfle dynamic library for client-side-encryption testing. Otherwise will rely on mongocryptd"
+   TRUE ENABLE_TESTS FALSE
+)
+
+if (MONGOC_TEST_USE_CSFLE)
+   find_program (_PYTHON3_EXE NAMES python3 py python)
```

Comment:
If I understand correctly, if `python3` is not found, then `py` or `python` may be used instead, either of which may be `python2` rather than `python3`. Is this acceptable?

Reply:
That's correct. That will end up causing an error and the fix would be similarly disabling csfle testing, although the error message won't be as pretty as when Python is completely absent. If we could use a newer CMake with a FindPython3 module, this would be a non-issue. Maybe someday…

Barely related aside pain: Windows now ships with a `python` command that just launches the Windows store page, so it's not even really possible to reliably tell whether the `python` executable you just found is even the real thing without trying to run a script with it, in which can you might accidentally open the Windows store page. Maybe they have some environment variable to suppress it, but still: pain.